src/main/java/ServerRunner.java

```diff
@@ -11,11 +11,15 @@
 public class ServerRunner {
 
     private static ExecutorService threadPool = Executors.newFixedThreadPool(20);
-    private static int port = 5000;
 
     public static void main(String[] args) {
+        int port = Integer.parseInt(args[1]);
+        String root = args[3];
         try {
-            HTTPServer server = new HTTPServer(new ServerSocket(port), new ThreadsExecutor(threadPool), new Logger());
+            HTTPServer server = new HTTPServer(new ServerSocket(port),
+                                               new ThreadsExecutor(threadPool),
+                                               new Logger(),
+                                               root);
```

Comment:
How about we create the `new FileSystem(root)` up here and pass it down to whoever needs it?
This way we can limit who needs to know about such little details as the `root` :)